app/views/admin/budget_headings/_errors.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id='error_explanation' class='callout alert'>
```

Comment:
It's best to have a single partial for errors
Try using this one, let's see if it fits nicely
https://github.com/consul/consul/blob/master/app/views/shared/_errors.html.erb